pages/overview/allocation-strategy.md

```diff
@@ -0,0 +1,23 @@
+---
+description: What role does a Voting Strategy play in Allo?
+---
+
+# Allocation Strategy
+
+Every [ pool ](/overview/pool) in Allo is managed by an allocation strategy,
+which determines:
+
+1. Recipients - Who is eligible to receive funding from the pool?
+2. Allocation - How will funds be allocated? (grants committee, token voting, SBTs)
```

Comment:
Suggest rephrasing this:

"Who is eligible to provide input on how the funds should be allocated, and how will they express their opinion?"